lib/systems/inspect.nix

```diff
@@ -91,7 +91,9 @@ rec {
     isMusl         = with abis; map (a: { abi = a; }) [ musl musleabi musleabihf muslabin32 muslabi64 ];
     isUClibc       = with abis; map (a: { abi = a; }) [ uclibc uclibceabi uclibceabihf ];
 
-    isEfi = [
+    # This is the list of platforms which can potentially have a UEFI firmware
+    # This is helpful to know when to emit UEFI binaries along your software, e.g. for boot or more.
+    hasEfi = [
```

Comment:
Perhaps we should have a deprecation warning for the old name?  I think it's plausible it's in use outside Nixpkgs.

Reply:
I would be surprised because I think most people would hardcode the result of the boolean (either build for UEFI or not) and rarely manage multiple platform firmwares outside of Nix.
Nevertheless, if we do a deprecation cycle, would you think a release cycle would be enough?

Reply:
AIUI, the gold standard is to introduce the new way, wait a release cycle, add a warning, wait a release cycle, and then remove the old way, so that external projects can always support both stable and unstable Nixpkgs without having to add conditionals.

But you're very probably right that nobody actually is going to be using this.

Reply:
Well, I can always do the gold standard and have `isEfiEnvironment` instead of `isEfi` and it's not the end of the world.

Reply:
Or just `isUefi`!

Reply:
Wouldn't this be _very_ confusing for anyone discovering that? :D

Reply:
Implemented deprecation warning and backward.

Reply:
We can move on the discussion on the new temporary name in the PR when I obtain the holy gnu-config triples for UEFI.